docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,103 @@
+}
+```
+
+You can make the values of the keys whatever makes sense for your use case, or you can use our example from our prototype below:
+
+![Setting up Eppo Feature Flag with JSON](/img/how-tos/integrating-with-webflow/setting-up-feature-flag.png)
+
+Once you have your feature flag setup, set your [feature flag up as an experiment](/experiments/creating-experiments). Once you have data flowing to your data warehouse from your feature flag, you will be able to analyze your results.
+
+## Setting up Webflow
+
+For the purposes of this tutorial, we will be very explicit on what things to test in your Webflow experiment. Keep in mind that all of the specific elements we’re testing are extendable to anything in your Webflow Environment.
+
+First, set up a Webflow landing page that has the following elements: a Header Title, a Header Subtitle, a Header CTA, and an image element. These elements should have no content in them for now since we will be filling in the image URL and messaging from our feature flag. As you are setting up these elements, make sure that you give each of the 4 elements one of the following custom ids: `header-subtitle`, `header-title`, `header-CTA`, and `header-image`. The custom code we add later specifically looks for these ids to write in the content from Eppo. See the screenshot below on where to update the ID for an element:
+
+![Setting up Webflow element IDs](/img/how-tos/integrating-with-webflow/setting-up-webflow-ids.png)
+
+Next, add the following code to your [custom code at the site level](https://university.webflow.com/lesson/custom-code-in-the-head-and-body-tags?topics=site-settings) in the footer:
+
+:::note
+
+By making sure our elements are “blank” to start with, we can completely reduce any flicker that would be caused by re-writing and element with Javascript. Make sure that you build in logic for default experience in the case that Eppo serves a `null` variation.
+
+:::
+
+```html
+<script>console.log('Loading eppo SDK')</script>
+<script src="https://cdn.jsdelivr.net/npm/@eppo/js-client-sdk@1.4.0/dist/eppo-sdk.js"></script>
+<script>console.log('Done loading eppo SDK')</script>
+
+<script>  
+// Init user parameters
+urlParams = new URLSearchParams(window.location.search);
+subject = urlParams.get('subject') || crypto.randomUUID();
+
+// Prep post-assignment code
+function setHeader() {
+  
+  // Get variation
+  const variationData = window.eppo.getInstance().getParsedJSONAssignment(subject, '<FEATURE-FLAG-KEY>');
+  console.log('>>> got variation data', variationData);
+  
+  // Add client side analytics tracking call to capture experiment data 
+  // and make sure assignment data is sent to your 
+  // data warehouse that is connected to Eppo.
+
+  document.getElementById('header-title').innerText = variationData.title;
+  document.getElementById('header-subtitle').innerText = variationData.subtitle;
+  document.getElementById('header-cta').querySelector('div').innerText = variationData.cta;
```

Comment:
One downside here is that we are listing every element here explicitly. Wonder if we can just loop over the variationData and set data directly (maybe this gets too complicated)

That is, in Python I would write this something like

```
for key, value in variationData.items():
    document.getElementById(key).innerText = value
```

Reply:
I think keeping it the way it is is slightly better for demonstration purposes. It's more explicit on what each thing is and where it's coming from. If people want to refactor, they can.

Reply:
In the larger vision ideally webflow elements have custom data attributes which common adapter code detects, and none of this custom js is needed!

e.g., `<img data-eppo-src-flag-key="headertest.image" />` or `<h1 data-eppo-text-flag-key="headertest.header></h1>`